src/install.ts

```diff
@@ -20,6 +20,9 @@ import * as path from "path";
  */
 export async function install(platform: string, architecture: string, release: string, products: string[]) {
     const releaseInfo = await matlab.getReleaseInfo(release);
+    if (releaseInfo.name < "r2020b") {
```

Comment:
What about using the version number instead?

Reply:
It seemed less straight forward to compare. You can't compare them as floats because there's multiple decimal points, and the lexicographic string ordering doesn't work because `"9.10.0" < "9.9.0" -> true`

Reply:
Is releaseInfo.name guaranteed to be lower?

Reply:
Yeah, the output of `getReleaseInfo` always outputs name as lower. It wouldn't be if we just took the `release` param the user gave us.